scripts/analytics/run_keen_counts.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import argparse
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+
+from keen.client import KeenClient
+
+from website.app import init_app
+from website.settings import KEEN as keen_settings
+from scripts.analytics.user_count import count as user_count
+from scripts.analytics.addon_count import count as addon_count
+from scripts.analytics.node_count import count as node_count
+from scripts.analytics.node_log_count import get_node_log_events as node_log_count
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_events_for_day(day):
+    logger.info('<---- Keen Counts for {} ---->'.format(day.isoformat()))
+
+    keen_events = {}
+    keen_events.update({'user_count_analytics': user_count(day)})
+    keen_events.update({'node_analytics': [node_count(day)]})
+    return keen_events
+
+def get_node_log_events(start, end):
+    events = node_log_count(start, end)
+    for i in range(0, len(events), 5000):
+        yield events[i:i + 5000]
+
+def main(start_date, end_date):
+    keen_project = keen_settings['private']['project_id']
+    write_key = keen_settings['private']['write_key']
+    if keen_project and write_key:
+        client = KeenClient(
+            project_id=keen_project,
+            write_key=write_key,
+        )
+    assert(client)
+    client.add_events({'addon_count_analytics': addon_count()})
```

Comment:
This should not be run if arguments were passed to the script (i.e. if an end_date and/or start_date is specified).